docs/source/06_nodes_and_pipelines/04_micro_packaging.md

```diff
@@ -1,17 +1,20 @@
+```eval_rst
+.. warning::  `kedro pipeline package/pull` will be deprecated in 0.18.0. We recommend using `kedro micropkg package` instead.
+```
 
 # Micro-packaging
 
-Micro-packaging allows users to share Kedro pipelines across codebases, organisations and beyond.
+Micro-packaging allows users to share Kedro micro-packages across codebases, organisations and beyond.
 
-## Package a modular pipeline
+## Package a micro-package
 
-You can package a modular pipeline by executing: `kedro pipeline package <pipeline_name>`
+You can package a micro-package by executing: `kedro micropkg package <pipeline_name>`
 
-* This will generate a new [wheel file](https://pythonwheels.com/) for this pipeline.
-* By default, the wheel file will be saved into `src/dist` directory inside your project.
+* This will generate a new [wheel file](https://pythonwheels.com/) for this micro-package.
```

Comment:
Just a heads-up: this will need to be updated to `tar file` when it's merged to `develop`, so keep an eye out on that when the automatic `main-to-develop` PR is created.